packages/electron-builder-squirrel-windows/src/SquirrelWindowsTarget.ts

```diff
@@ -98,24 +109,35 @@ export default class SquirrelWindowsTarget extends Target {
     const appInfo = packager.appInfo
     const projectUrl = await appInfo.computePackageUrl()
     const appName = this.appName
+    // If not specified will use the Squirrel.Windows that is shipped with electron-installer(https://github.com/electron/windows-installer/tree/main/vendor)
+    // After https://github.com/electron-userland/electron-builder-binaries/pull/56 merged, will add `electron-builder-binaries` to get the latest version of squirrel.
+    const vendorDirectory = this.options.customSquirrelExePath
```

Comment:
We need to check this. Also `customSquirrelExePath` is different than a `vendorDirectory`, so we should rename the variable as well:
```
    let vendorDirectory = this.options.customSquirrelVendorDir
    if (isEmptyOrSpaces(vendorDirectory) || !(await exists(vendorDirectory))) {
      log.warn({ vendorDirectory }, "unable to access Squirrel.Windows vendor directory, falling back to default electron-winstaller")
      vendorDirectory = undefined
    }
```

Reply:
Fixed.